pgbadger

```diff
@@ -15247,6 +15259,18 @@ sub skip_unwanted_line
 		return 1 if ($found);
 	}
 
+	if ($#include_time >= 0) {
+		my $found = 0;
+		foreach (@include_time) {
+			if ($prefix_vars{'t_timestamp'} !~ /$_/) {
+				$found = 1;
+				last;
+			}
+        }
```

Comment:
I'm not a perl monk, but this brace looks lost in the code. Is there mixed tabs and spaces ? Actually, that's only cosmetic. :-)